spec/class.dd

```diff
@@ -1203,6 +1203,76 @@ $(P When a scope class reference goes out of scope, the destructor (if any) for
 it is automatically called. This holds true even if the scope was exited via a
 thrown exception.)
 
+
+$(H2 $(LNAME2 abstract, Abstract Classes))
+
+$(P
+        An abstract member function must be overridden by a derived class.
+        Only virtual member functions may be declared abstract; non-virtual
+        member functions and free-standing functions cannot be declared
+        abstract.
+)
+
+$(P
+        A class is abstract if any of its virtual member functions
+        are declared abstract or if they are defined within an
+        abstract attribute.
+        Note that an abstract class may also contain non-virtual member functions.
```

Comment:
I suppose you merged the two paragraphs because the following example applies to both of them.